flathub.json

```diff
@@ -1,3 +1,3 @@
 {
-  "only-arches": ["x86_64"]
```

Comment:
i think you could remove this altogether since these are the two arches that are build in flathub anyhow at the moment, but not important, let's merge this and if we feel like removing this file, we can do it in a separate PR